gapic/templates/docs/index.rst.j2

```diff
@@ -2,6 +2,16 @@ API Reference
 -------------
 .. toctree::
     :maxdepth: 2
-    
+
+    {% set index_ns = namespace(has_services_api=false) %}    
+    {% for service in api.services.values() | selectattr('name') %}
+    {% if service.name == "Services" %}
```

Comment:
Should we also check for `service.name == "Types"` and set `index_ns.has_types_api`?